hibernate-community-dialects/src/test/java/org/hibernate/community/dialect/AltibaseFunctionsTest.java

```diff
@@ -10,7 +10,9 @@
 import org.hibernate.testing.orm.junit.RequiresDialect;
 import org.hibernate.testing.orm.junit.SessionFactory;
 import org.hibernate.testing.orm.junit.SessionFactoryScope;
-import org.junit.jupiter.api.*;
```

Comment:
spotless can not handle wildcards (properly).

- https://github.com/diffplug/spotless/pull/2526
- https://github.com/diffplug/spotless/pull/2571